packages/@glimmer/manager/lib/internal/defaults.ts

```diff
@@ -0,0 +1,44 @@
+import { buildCapabilities } from '../util/capabilities';
+
+import type { CapturedArguments as Arguments, HelperCapabilities } from '@glimmer/interfaces';
+
+type FnArgs<Args extends Arguments = Arguments> =
+  | [...Args['positional'], Args['named']]
+  | [...Args['positional']];
+
+type AnyFunction = (...args: any[]) => unknown;
+
+interface State {
+  fn: AnyFunction;
+  args: Arguments;
+}
+
+export class FunctionHelperManager {
```

Comment:
```suggestion
export class FunctionHelperManager implements HelperManagerWithValue<State> {
```

Otherwise we can make a mistake in here (or the requirements might have changed), but you will get the error in a very far away location (probably where you try to pass it into the constructor of `CustomHelperManager`)